### ISSUE TITLE
FlashLoanReceiverBase.sol : TypeError: Public state variable has override specified but does not override anything.

### ISSUE THREAD

**Issue by saad-121** [aave/protocol-v2#239]
I am getting errors re these two lines on the FlashLoanReceiverBase.sol:

```
  ILendingPoolAddressesProvider public immutable override ADDRESSES_PROVIDER;
  ILendingPool public immutable override LENDING_POOL;
```

as it is not clear what they are overriding. There are no matching variable names on the IFlashLoanReceiver interface.

When I try to truffle compile, I get this error for both lines:

`TypeError: Public state variable has override specified but does not override anything.`

I am tempted to  just remove the override but don't want to do something that may have dangerous repercussions. 

If I do remove the overrides (just for testing), I get this error about the constructor:

`DeclarationError: Abstract contracts cannot have public constructors. Remove the "public" keyword to fix this.`

Assistance on these errors would be appreciated. Thank you.

**Comment by atang08** [comment]
have this issue too, if i figure something out, i will try to update this.

**Comment by atang08** [comment]
yeah, I just ended up removing public and override and error is gone.    hopefully someone can have a better solution to this.
